src/dsp_tools/commands/xmlupload/models/serialise/jsonld_serialiser.py

```diff
@@ -0,0 +1,35 @@
+import json
+from typing import Any
+
+from pyld import jsonld
+from rdflib import Graph
+
+
+def serialise_property(rdf_graph: Graph, prop_name: str) -> dict[str, Any]:
+    """
+    This function makes the serialised json-ld conform to the format expected by the DSP-API.
```

Comment:
No it is an rdf graph, there are two steps to serialise it so that the api accepts it.
The api does not accept all json-ld formats (unfortunately there are variations). We cannot force rdflib to make the json-ld in the format we want, but in a second step we can change the format. Because the serialisation itself needs a little transformation, I decided that this function should take a graph and not a json-ld.

Reply:
But then the docstring is not correct. The input is a graph, but the docstring suggests that the input is a "serialised json-ld"